app/controllers/admin/tags_controller.rb

```diff
@@ -28,7 +29,7 @@ def create
       @tag.name = params[:tag][:name].downcase
 
       if @tag.save
-        flash[:success] = "Tag has been created!"
+        flash[:success] = "#{@tag.name} has been created!"
```

Comment:
We do this in the `Admin::TagsController#update` method and when adding a moderator to a tag (but we display the moderator's name rather than the tag name!), so I updated this for consistency.